core/block_svg.js

```diff
@@ -119,6 +119,14 @@ Blockly.BlockSvg.prototype.width = 0;
 Blockly.BlockSvg.prototype.dragStartXY_ = null;
 
 /**
+ * Map from IDs for warnings text to PIDs of functions to apply them.
+ * Used to be able to maintain multiple warnings.
+ * @type {Object<string, number>}
+ * @private
+ */
+Blockly.BlockSvg.prototype.warningTextDb_ = null;
```

Comment:
do we need to dispose of this now in the destructor?

Reply:
Yes.  Good call.

Reply:
Updated, PTAL.